### PR TITLE
Run all evals as jobs within a single Beaker experiment

### PR DESCRIPTION
**Pull request by dwadden**
I find it convenient to have all evals for a given model associated with a single [experiment](https://beaker.org/ex/01HQQ5VX550JASBSF634JV1ZXQ/tasks/01HQQ5VX5AS4M7XCDK6GMF8NCT/job/01HQQ5XZ41FWR2RDVX12N2C8TC).

This PR implements this. It also has the added side effect only creating 1 new template `yaml` instead of `n_tasks` of them. I'll comment the code below to explain what's happening.

I've tested this (for instance the beaker job above) and it seems to work. This is also purely personal preference, so if you don't like this we can just make it a separate script called `submit_eval_experiment` or something; whatever you guys prefer.

### CODE REVIEW

File: scripts/submit_eval_jobs.py [hunk 1]
```diff
@@ -56,32 +56,27 @@
...
-models = [
-    # llama1 models
-    # ("llama1-7B", "01HCCBK1MYKXKQC0C6CSVW1F22", None, "vanilla_lm"),
-
-    # other causal models
-    # ("hf-opt-7B", "facebook/opt-6.7b", None, "vanilla_lm"),
-    # ("finetuned_opt", "01H13EBXSADXXJCRERART90ZKJ", None, "tuned_lm"),
-    (args.model_name, args.location, args.beaker_subfolder, model_type),
-]
+model_info = (args.model_name, args.location, args.beaker_subfolder, model_type)
```
Comment (hamishivi) [inline scripts/submit_eval_jobs.py:61]: Could you leave the comments for hf/beaker/etc examples? Just to make it extra clear what these arguments should be.

Comment (hamishivi) [reply scripts/submit_eval_jobs.py:61]: We also lose the ability to batch-submit models (via the list) here, but this is probably fine.

File: scripts/submit_eval_jobs.py [hunk 2]
```diff
@@ -56,32 +56,27 @@
...
 
 #--------------- experiments about number of supervision tasks -------------------------
 
 # for experiment_group, model_info in itertools.product(experiment_groups, models):
-for model_info, experiment_group in itertools.product(models, experiment_groups):
+
+d = copy.deepcopy(d1)
```
Comment (hamishivi) [inline scripts/submit_eval_jobs.py:67]: Did you alter the default yaml config at all? I'll check this in running this myself, but just making sure.

**Review by hamishivi — COMMENTED** [review]
Seems good to me! Will approve and merge once I get the chance to run it myself.

As a note, you can restart/cancel tasks in an experiment independent of the other tasks, right? e.g. if an eval task needs to be restarted or rerun.

**Comment by dwadden** [comment]
You can cancel jobs independently. To do this, you click on the job you way to cancel, copy the hash after the last slash in the URL (that's the job id), and do `beaker job cancel [job-id]`.

You can only restart at the experiment level rather than job level. By default, when you call `beaker experiment resume [experiment-id]`, Beaker will check for any failed or stopped jobs in the experiment and re-run them, which seems reasonable? We could ask the beaker team to add functionality to restart individual jobs if you think this is important though.

**Comment by hamishivi** [comment]
Do you mean that if you hit restart, it only restarts jobs that have failed or were cancelled? That's fine for now, just making sure that you don't rerun everything if one task fails for an ephemeral reason (e.g., RAM issues).

**Comment by dwadden** [comment]
Correct, it only restarts failed or canceled jobs and leaves the completed ones untouched.

**Review by hamishivi — APPROVED** [review]
Tested and works.

**Pull request merged**